public/apps/login/test/login-page.test.tsx

```diff
@@ -31,20 +70,61 @@ describe('Login page', () => {
   };
 
   describe('renders', () => {
-    it('renders with config value', () => {
-      const config: ClientConfigType['ui']['basicauth']['login'] = {
-        title: 'Title1',
-        subtitle: 'SubTitle1',
-        showbrandimage: true,
-        brandimage: 'http://localhost:5601/images/test.png',
-        buttonstyle: 'test-btn-style',
+    it('renders with config value: string array', () => {
+      const config: ClientConfigType = {
+        ui: configUI,
+        auth: {
+          type: ['basicauth'],
```

Comment:
Could we re-use the constant in all places? There're quite a few occurrences that are hardcoded.

Reply:
I already declare the constant for configUI, which is used multiple times. But for auth type, it is different for every test case. So I would recommend to maintain the same.

Reply:
We can unify all the hardcoded values when resolving this issue: https://github.com/opensearch-project/security-dashboards-plugin/issues/1160